packages/cardano-services/src/Program/options/policyIds.ts

```diff
@@ -0,0 +1,27 @@
+import { Cardano, NotImplementedError } from '@cardano-sdk/core';
+import { Command, Option } from 'commander';
+
+const handlePolicyIdsParser = (policyIds: string) => policyIds.split(',').map(Cardano.PolicyId);
+
+export enum HandlePolicyIdsOptionDescriptions {
+  HandlePolicyIds = 'Handle policy ids'
+}
+
+export interface HandlePolicyIdsProgramOptions {
+  handlePolicyIds: Cardano.PolicyId[];
+}
+
+export const withHandlePolicyIdsOptions = (command: Command, maxPolicyIdsSupported?: number) =>
+  command.addOption(
+    new Option('--handle-policy-ids <handlePolicyIds>', HandlePolicyIdsOptionDescriptions.HandlePolicyIds)
+      .env('HANDLE_POLICY_IDS')
+      .argParser((policyIds: string) => {
+        const policyIdsArray = handlePolicyIdsParser(policyIds);
+
+        if (maxPolicyIdsSupported && policyIdsArray.length > maxPolicyIdsSupported) {
+          throw new NotImplementedError('Multiple policyIds not supported');
```

Comment:
```suggestion
          throw new NotImplementedError(`${policyIdsArray.length} policyIds are not supported`);
```

Reply:
aa8096909c8ee4b312f8d2d9395b28766ebd4ff2 and af7ef863a0e74c879dd5daba73879a75a9f60662 should solve this